ember-file-upload/src/queue.ts

```diff
@@ -226,6 +230,8 @@ export class Queue {
         element.removeEventListener('change', changeHandler);
       };
     },
+    // @ts-expect-error: ember-modifier@^3 requires an options hash as second argument
+    // used to opt-in to lazy argument handling, which is the default for ember-modifier@^4
     { eager: false }
```

Comment:
Not sure if using `@ts-expect-error` is the correct approach here. I covered that one in the Discord question as well.

Reply:
I guess the other way would be to detect the `ember-modfier` version with embroider macros. As long as this actually works on both versions I think that's fine.

Just a thought – should we add ember-try scenarios to ensure both versions of `ember-modfiier` work?

Reply:
> I guess the other way would be to detect the `ember-modfier` version with embroider macros. As long as this actually works on both versions I think that's fine.

I actually tried that. TypeScript even complains if passing `undefined`. So we would need to duplicate entire function call. I'm not sure if that would be a good trade-off just for making TypeScript happy.

Reply:
I agree 👍🏻